explainer.md

```diff
@@ -1,3 +1,118 @@
+
+The solution proposed in this document is based on a new RTP header extension, namely absolute capture time, which contains two data fields:
+ * an absolute capture timestamp, which is stamped by the original capturer, and is supposed to be received untouched by the end receivers.
+ * an estimated clock offset with respect to the capturer's clock, which is supposed to be read and updated by every intermediate sender.
+
+With the absolute capture timestamps, end receivers can accurately measure the audio-video synchronization performance. With the `estimated clock offset`, which gets updated at each intermediate hop, end receivers can estimate their respective clock offset with respect to the capturer's clock, and then together with the absolute capture timestamp, measure the end-to-end delay. 
+
+The absolute capture time RTP header extension has been adopted by WebRTC, see the specification [here](https://github.com/webrtc/webrtc-org/blob/gh-pages/experiments/rtp-hdrext/abs-capture-time/index.md).
+
+To surface the information to web applications, two data fields are proposed to be added to RTCRtpContributingSource, and has been adopted by WebRTC 1.0 extensions, see the specification [here](https://w3c.github.io/webrtc-extensions/#rtcrtpcontributingsource-dictionary).
+
+#### Goals
+
+ * Facilitate calculation of audio video synchronization performance in real-time communication systems.
+ * Facilitate calculation of end-to-end delay in real-time communication systems, particularly those that involve intermediate stream regenerators.
+
+##### Non-goals
+
+This proposal does not aim for improving the accuracy of end-to-end delay measurement in real-time communication systems that can already measure it based on RTCP based schemes.
+
+#### [API 1]: captureTimestamp
+
+We propose to add captureTimestamp, type of DOMHighResTimeStamp, to the RTCRtpContributingSource dictionary. This surfaces the absolute capture timestamp in the absolute capture time RTP header extension, when it is present or can be extrapolated from previously received data, for the last rendered audio or video frame. It can be used for measuring audio video synchronization performance as illustrated in the following exemplar code:
+
+    [receiverAudio, receiverVideo] = peerconnection.getReceivers();
+
+    latestCaptureTimestampsAudio = receiverAudio.getSynchronizationSources()[0].captureTimeStamp;
+    latestCaptureTimestampsVideo = receiverVideo.getSynchronizationSources()[0].captureTimeStamp;
+
+    avSynchronization = latestCaptureTimestampsAudio - latestCaptureTimestampsVideo;
+
+This new API has been accepted in WebRTC 1.0 extensions. See the specification [here](https://w3c.github.io/webrtc-extensions/#dom-rtcrtpcontributingsource-capturetimestamp).
```

Comment:
The explainer of the spec that has accepted the feature does not need to say that it has been accepted by itself. Just say that "captureTimestamp is defined here" if you think a link is needed.